.github/FUNDING.yml

```
@@ -1,3 +1,6 @@
 # These are supported funding model platforms
 
 github: nataliagranato
+ko_fi: nataliagranato
+buy_me_a_coffee: nataliagranato
+pix: fada126d-7311-495f-bda2-55ec79b0a39b
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)